README.md

```diff
@@ -262,6 +274,29 @@ FIRETWEET_MAIN_DIR=/path/to/firetweet/src/main make android-lib
 You can also override this environment variable if you want to use the
 [Flashlight Android Tester](https://github.com/getlantern/lantern-mobile-single-app-example) app.
 
+#### Creating the Android library without docker
+
+1. Install Java JDK 7 or 8
+2. Install [Android SDK Tools](http://developer.android.com/sdk/index.html#Other)
+3. Install NDK(http://developer.android.com/ndk/downloads/index.html)
+4. Install lantern's [fork of gomobile](https://github.com/getlantern/mobile)
```

Comment:
Looks like I might have a clang from homebrew or something:

```
clang version 3.3 (trunk 179269)
Target: x86_64-apple-darwin15.2.0
Thread model: posix
```

Let me try uninstall that and see what happens.